articles/identity-labs/01-web-sign-in/exercise-01.md

```diff
@@ -28,14 +28,17 @@ A simple Node.js Express application has been created to get you started. This i
 
 The following steps will show you how to set up and run the web application that you will be securing:
 
-1. Go to the `/lab-01/begin` folder in your local copy of the [identity exercise repo](https://github.com/auth0/identity-102-exercises/).
+1. Open your Terminal app, clone the [identity exercise repo](https://github.com/auth0/identity-102-exercises/), then go to the `/lab-01/begin` folder:
 
-::: note
-If you've never cloned a repo before, please see [GitHub's instructions ](https://help.github.com/en/articles/cloning-a-repository).
-:::
+```bash
+❯ git clone git@github.com:auth0/identity-102-exercises.git
+Cloning into 'identity-102-exercises'...
+
+❯ cd identity-102-exercises/lab-01/begin
+```
 
-2. Review the `server.js` code. This is a generic Node.js HTTP server that uses `body-parser` to parse the JSON, buffer, string, and URL-encoded data received. It also includes `morgan` to log HTTP requests.
-3. The `.env-sample` file will be used for the environment variables you need for this lab. It’s populated with `PORT` (the port number where the app will run). You will set the rest of the values later on in the lab. For now, create a copy of the file in the same folder and name it `.env`. Run the following commands in your terminal:
+2. Open a code editor like VS Code or Atom in the same directory (File > Open) and review the `server.js` code. This is a generic Node.js HTTP server that uses Express's built-in `urlencoded` to parse the JSON, buffer, string, and URL-encoded data received. It also includes `morgan` to log HTTP requests.
```

Comment:
Maybe stick the lab instructions to VS Code for clarity. People who have their own editor will just use it without instruction.

Reply:
I'm starting to lean that direction. There are a lot of ways this could be simplified to keep people moving. Wish I had a bit more time to refine it 🤷‍♂